libs/ui/packages/shared/index.ts

```diff
@@ -1 +1,4 @@
+export * from "./src/assorted/Icon/Providers";
+export * from "./src/assorted/Icon/Sizes";
+export * from "./src/consts/urls";
```

Comment:
Please don't put the provider icon logic in UI-Shared as it's reserved for the Design System. We want to get rid of the Provider Icons in the Design System. UI Shared is reserved for design system design tokens.
I'd recommend to have that in LLC instead of UI-Shared.

Reply:
@mle-gall Thanks for the feedback. I've moved this code to LLC instead of UI-shared